src/Controller/AddOnOptionsController.py

```diff
@@ -79,6 +83,13 @@ def __init__(self, window):  # initialization function
                 "parent_ID": 442,
                 "short_name": "Line & Fill configuration",
             },
+            {"level": 0, "dbID": 446, "parent_ID": 6, "short_name": "Configuration"},
```

Comment:
While the mapping of these magic numbers is obvious, the origin of the specific values is not.  Are these constants in use elsewhere? Even if the values origins were arbitrary it seems like these might be shared and hence should be defined as constants in a shared module.

Reply:
These magic numbers are already in the previous version of onkoDICOM, we just added the tab for Configuration. 
They are not used in anywhere else.